docker-compose-base.yml

```diff
@@ -216,23 +216,51 @@ services:
     environment:
       ZOOKEEPER_CLIENT_PORT: 32181
       ZOOKEEPER_TICK_TIME: 2000
+      ZOOKEEPER_INIT_LIMIT: 5
+      ZOOKEEPER_SYNC_LIMIT: 2
+      ZOOKEEPER_SESSION_TIMEOUT_MS: 750  # may be too short for prod.
```

Comment:
Could you suggest a better value for prod? Also, could you add some comments for the other options - or do we have our preferred values documented somewhere?

Reply:
SYNC and INIT are defaults so they could be removed. 

Session timeout would depend on latency of the production network. On local, it's about 0.5ms. 750 is likely a fine value in prod, but there's no way to know at this point. If it's too short, ZK will close Kafka session when it shouldn't, which is suboptimal but not problematic. Having it set too long causes problems if Kafka goes down, restart and then ZK expects it to still be taking part in the old session.